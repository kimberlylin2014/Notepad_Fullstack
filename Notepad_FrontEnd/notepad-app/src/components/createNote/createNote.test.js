import { shallow } from 'enzyme';
import React from 'react';
import CreateNote from './createNote.component';

it('expect to render Create Note component', ()=> {
    const mockButtonLabel = 'Create';
    expect(shallow(<CreateNote buttonLabel={mockButtonLabel}/>)).toMatchSnapshot();
})

it("correctly rendering OnClick",() => {
    const mockButtonLabel = 'Create';
    const wrapper = shallow(<CreateNote buttonLabel={mockButtonLabel}/>);
    wrapper.find('[id="createNote"]').simulate('onChange')
})


it("correctly rendering OnSubmit",() => {
    const mockButtonLabel = 'Create';
    const wrapper = shallow(<CreateNote buttonLabel={mockButtonLabel}/>);
    wrapper.find('[id="createNote"]').simulate('onSubmit')
})