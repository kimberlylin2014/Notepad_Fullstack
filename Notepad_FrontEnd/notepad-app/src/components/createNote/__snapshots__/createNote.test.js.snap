// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Create Note component 1`] = `
<div
  className="CreateNote container"
>
  <CreateNoteModal
    buttonLabel="Create"
    handleSubmit={[Function]}
    id="createNote"
    onChange={[Function]}
  />
</div>
`;
